core/sources/domains.move

```diff
@@ -71,6 +71,14 @@ module aptos_names::domains {
         registry: Table<NameRecordKeyV1, NameRecordV1>,
     }
 
+    struct ReverseLookupRegistryV1 has key, store {
```

Comment:
Can you add more documentations for these structs?

Reply:
Done